src/zenhub.py

```diff
@@ -69,9 +79,107 @@ def _get_repo_id(repo_name):
         try:
```

Comment:
Maybe this could simplified by making the keys in repo be 'azul', 'sync-agile-boards', 'sync-test', etc. so we don't have to check each one? e.g. try: return str(repo[repo_name]) except KeyError as err: